src/Superpower/Superpower.csproj

```diff
@@ -1,9 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Condition=" '$(OS)' != 'Windows_NT' ">
+    <TargetFrameworks>netstandard1.0;netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(OS)' == 'Windows_NT' "> 
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
```

Comment:
Need to include netstandard1.0 here or we'll have a compatibility regression in the package (currently 1.0 lets the NUPKG run everywhere).

Reply:
The new library guidance from MS says that you should just target netstandard2.0 and the full frameworks and avoid netstandard1.x

https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting

Reply:
If we drop netstandard1.x support here we'd need to rev the major version number, as it'd be a breaking change for [several older platforms](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support) that the library already supports.

The guidance makes some concession to this:

> You should only target .NET Standard 1.x if you specifically need to target an older platform.

..which SemVer requires in this case. Perhaps we earmark this for the next major version of this library?

Reply:
I added netstandard1.0 back.